server/config.ts

```diff
@@ -133,5 +133,6 @@ export default {
     plpPrisonerListAndOverviewPagesEnabled: toBoolean(
       get('PLP_PRISONER_LIST_AND_OVERVIEW_PAGES_ENABLED', false, requiredInProduction),
     ),
+    timelinePageEnabled: toBoolean(get('TIMELINE_PAGE_ENABLED', false, requiredInProduction)),
```

Comment:
Nice 👍 
But I think you _might_ need to add `TIMELINE_PAGE_ENABLED` to the main helm `values.yaml` file (because of the `requiredInProduction` flag, which actually means "required in a deployed environment")
I suspect that if you dont add it to the helm stuff it wont deploy and startup in `dev`

(TBH we should add it anyway for completeness 👍 )

Reply:
Added this

Reply:
May just need re-approval as I've dismissed your review with this change @nathanrussell-moj-digital